test/intl402/NumberFormat/constructor-max-and-min-exponent-limits.js

```diff
@@ -0,0 +1,19 @@
+// Copyright 2023 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-tointlmathematicalvalue
+description: Tests exponent limits
+features: [Intl.NumberFormat-v3]
+---*/
+
+const nf = Intl.NumberFormat();
+
+assert.sameValue(nf.format(-100e309), "-∞", "Should output -∞");
```

Comment:
Request: please test values that are _slightly below_, _exactly equal to_, and _slightly above_ the cutoff points.